doc/architecture/platforms/snp.rst

```diff
@@ -0,0 +1,28 @@
+AMD SEV-SNP
+===================
+
+.. warning:: SEV-SNP support is currently experimental and in active development.
+
+How to use the AMD SEV-SNP platform
+-----------------------------------
+CCF must run on an AMD CPU which supports SEV-SNP.
+
+To use SNP, ``virtual`` must be specified in ``COMPILE_TARGETS``, at compile time. If any other targets are specified, you may also need to specify the ``TEST_ENCLAVE=virtual`` runtime argument.
+
```

Comment:
Are there plans to support a new `COMPILE_TARGET` option for AMD servers? If not, should we just combine the below `Insecure Virtual` section to this? Basically clarifying that `virtual` target will run on any platform and therefore for AMD, we select `virtual`

Reply:
Thanks for the issue link. I didn't mean to remove the AMD page. We're saying:

`To use SNP, ``virtual`` must be specified in ``COMPILE_TARGETS``, at compile time.`
and also,
`To use the insecure virtual platform, ``virtual`` must be specified in ``COMPILE_TARGETS`` at compile time.`

I was wondering if the "insecure virtual platform" can be merged up into the AMD page in some form. But I agree that it's easier to discover them both in the current format you have.